testing/it/e2e-selenium/src/test/java/com/adobe/cq/wcm/core/components/it/seljup/tests/embed/v1/EmbedIT.java

```diff
@@ -43,7 +43,7 @@
 public class EmbedIT extends AuthorBaseUITest {
 
     private static String componentName = "embed";
-    private static String youtubeEmbedField = "core/wcm/components/embed/v1/embed/embeddable/youtube";
+    private static String youtubeEmbedField = "core/wcm/components/embed/v1/embed/embeddable/youtube/v1";
```

Comment:
We should add specific IT tests for `embed/v1/embed/embeddable/youtube/v2/youtube`